cmd/csaf_checker/processor.go

```diff
@@ -1217,7 +1217,9 @@ func (p *processor) checkProviderMetadata(domain string) bool {
 
 	for i := range lpmd.Messages {
 		// TODO: Filter depending on the role.
```

Comment:
The TODO should be removed.

Reply:
Done via https://github.com/csaf-poc/csaf_distribution/pull/429/commits/16041bdb897aa9208f2c438cb0a521e7d3337b53